src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinatorModel.swift

```diff
@@ -410,4 +414,73 @@ class ExposureSubmissionCoordinatorModel {
 			fatalError("Cannot set submission consent, no corona test type is set")
 		}
 	}
+
+	/// Check SRS Flow Prerequisites.
+	/// - Parameters:
+	/// 	- isLoading: The callback that should be execute while fetching the self service parameters from app configuration.
+	/// 	- completion: The completion handler, that has given the regarding `SRSPreconditionError`, in case of a precondition fault.
+	func checkSRSFlowPrerequisites(
+		isLoading: @escaping CompletionBool,
+		completion: @escaping (Result<Void, SRSPreconditionError>) -> Void
+	) {
+		// Check if app was installed less than 48 hours
+		guard
+			let appInstallationDate = store.appInstallationDate,
```

Comment:
lets have a call for this

Reply:
like discussed, thank you for the creative flow :)